package.json

```diff
@@ -30,7 +30,7 @@
     "test:setup": "node tools/config.js",
     "pretest:saucelabs": "lerna run --parallel pretest",
     "test:saucelabs": "karma start config/karma.saucelabs.js --single-run",
-    "docgen": "node scripts/docgen/generate-docs.js"
+    "docgen": "node scripts/docgen/generate-docs.js --api js; node scripts/docgen/generate-docs.js --api node"
```

Comment:
I think it'd be helpful to be able to run js or node separately. So I'd suggest to add 2 more scripts:
```javascript
{
"docgen:js": ...
"docgen:node": ...
"docgen": "npm run docgen:js; npm run docgen:node"
}
```

Reply:
Done.